src/components/FindMovie/FindMovie.js

```diff
@@ -1,55 +1,107 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 import './FindMovie.scss';
-
 import { MovieCard } from '../MovieCard';
-import movies from '../../api/movies.json';
-
-export const FindMovie = () => (
-  <>
-    <form className="find-movie">
-      <div className="field">
-        <label className="label" htmlFor="movie-title">
-          Movie title
-        </label>
-
-        <div className="control">
-          <input
-            type="text"
-            id="movie-title"
-            placeholder="Enter a title to search"
-            className="input is-danger"
-          />
-        </div>
+import { getResponse } from '../../api/findMovie';
 
-        <p className="help is-danger">
-          Can&apos;t find a movie with such a title
-        </p>
-      </div>
+export const FindMovie = ({ onAddMovie }) => {
+  const [title, setTitle] = useState('');
+
+  const [movie, setMovie] = useState(null);
+
+  const [movieFound, setMovieFind] = useState(true);
+
+  useEffect(() => {
+    setMovieFind((true));
+  }, [movie]);
 
-      <div className="field is-grouped">
-        <div className="control">
-          <button
-            type="button"
-            className="button is-light"
-          >
-            Find a movie
-          </button>
+  const onFindMovie = () => {
+    getResponse(title).then((result) => {
+      if (!result.Error) {
+        setMovie({
+          title: result.Title,
+          imgUrl: result.Poster,
+          description: result.Plot,
+          imdbId: result.imdbID,
+          imdbUrl: `https://www.imdb.com/title/${result.imdbID}`,
+        });
+      } else {
+        setMovieFind(false);
+      }
+    });
+  };
+
+  return (
+    <>
+      <form className="find-movie">
+        <div className="field">
+          <label className="label" htmlFor="movie-title">
+            Movie title
+          </label>
+
+          <div className="control">
+            <input
+              type="text"
+              id="movie-title"
+              placeholder="Enter a title to search"
+              className={classNames({
+                input: true,
+                'is-danger': !movieFound,
+              })}
+              value={title}
+              onChange={(event) => {
+                setTitle(event.target.value);
+              }}
+            />
+          </div>
+          {movieFound || (
```

Comment:
```suggestion
          {!movieFound &&(
```
Use `&&` for conditional rendering